snapcraft/plugins/qml.py

```diff
@@ -72,7 +72,7 @@ def _build_qt_config(self):
         config = open(os.path.join(configdir, 'snappy-qt5.conf'), 'w')
         config.write('./usr/lib/{}/qt5/bin\n'.format(arch))
         config.write('./usr/lib/{}\n'.format(arch))
-        config.close
+        config.close()
```

Comment:
Huh, this must have sneaked by review. The fix should really be using a `with` though-- mind implementing that instead?